docs/buildah-config.md

```diff
@@ -82,6 +82,12 @@ Note: this setting is not present in the OCIv1 image format, so it is discarded
 Add an image *label* (e.g. label=*value*) to the image configuration of any
 images which will be built using the specified container. Can be used multiple times.
 
+**--onbuild** *onbuild command*
+
+Add an ONBUILD command to the image.  ONBUILD commands are automatically run
+when images are built based on the image you are creating.  ONBUILD images are
+only supported on `docker` formatted images.
+
```

Comment:
Might be nice to have an example for OnBuild.  Would it be good to add a note here to show how to determine what type of container it is?   A doc somewhere to show the difference between OCI and Docker formatted images and the commands applicable to each would be good.  Trello card for that?

Reply:
Nope, I missed adding the variable to the exec too.  Back to testing.